framework/branch-key-store.md

```diff
@@ -870,36 +846,36 @@ A branch key record MUST include the following key-value pairs:
 1. `hierarchy-version`: Version of the hierarchical keyring;
    represented as [AWS DDB Number](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html#HowItWorks.DataTypes)
 
-A branch key record MAY include [custom encryption context](#custom-encryption-context) key-value pairs.
-These attributes MUST always be prefixed with `aws-crypto-ec:` regardless of the item's `hierarchy-version`.
+A branch key record MAY include [encryption context](./structures.md#encryption-context) key-value pairs.
+These attributes MUST be always prefixed with `aws-crypto-ec:` regardless of the item's `hierarchy-version`.
 
-### Branch Key Materials From Authenticated Encryption Context
+### Branch Key Materials From Authenticated Branch key Context
 
 The `type` attribute MUST either be equal to `"branch:ACTIVE"` or start with `"branch:version:"`.
 
 If the `type` attribute is equal to `"branch:ACTIVE"`
-then the authenticated encryption context MUST have a `version` attribute
+then the authenticated branch key context MUST have a `version` attribute
 and the version string is this value.
 If the `type` attribute start with `"branch:version:"` then the version string MUST be equal to this value.
 
-To construct [branch key materials](./structures.md#branch-key-materials) from authenticated encryption context as follows:
+To construct [branch key materials](./structures.md#branch-key-materials) from authenticated branch key context as follows:
 
 - [Branch Key](./structures.md#branch-key) MUST be the [decrypted branch key material](#aws-kms-branch-key-decryption)
 - [Branch Key Id](./structures.md#branch-key-id) MUST be the `branch-key-id`
 - [Branch Key Version](./structures.md#branch-key-version)
   The version string MUST start with `branch:version:`.
   The remaining string encoded as UTF8 bytes MUST be the Branch Key version.
 - [Encryption Context](./structures.md#encryption-context-3) MUST be constructed by
-  [Custom Encryption Context From Authenticated Encryption Context](#custom-encryption-context-from-authenticated-encryption-context)
+  [Encryption Context From Authenticated Branch Key Context](#encryption-context-from-authenticated-branch-key-context)
 
-### Custom Encryption Context From Authenticated Encryption Context
+### Encryption Context From Authenticated Branch Key Context
```

Comment:
I think that this is a really good section. I think you can use this everywhere.